cos/models/Game.py

```diff
@@ -42,6 +42,7 @@ def __init__(self, name):
         self.turn_order = []
         self.turn_cycle = None
         self.game_started = False
+        self.winner = "none"
```

Comment:
This should be None, not "none". Python has the concept of a null object called None. Also it Python doesnt care about typing, so setting it to a None object and then changing it to a string later is perfectly fine.

Reply:
That's perfect. I wasn't aware of that. Will update.